test/compound/setup/TestSetupVaults.sol

```diff
@@ -9,12 +9,16 @@ import {SupplyVault} from "@vaults/compound/SupplyVault.sol";
 
 import "@morpho-labs/morpho-utils/math/PercentageMath.sol";
 
+import "../../helpers/interfaces/IRolesAuthority.sol";
 import "../../helpers/FakeToken.sol";
 import "../helpers/VaultUser.sol";
 
 contract TestSetupVaults is TestSetup {
     using SafeTransferLib for ERC20;
 
+    address internal constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+    address internal constant MORPHO_TOKEN = 0x9994E35Db50125E0DF82e4c2dde62496CE330999;
```

Comment:
Maybe it is more convenient to have an `ERC20`

Reply:
Not really since the address version is used many times too now...